thoth/storages/inspection_schema.py

```diff
@@ -74,10 +61,10 @@
 INSPECTION_SCHEMA = Schema(
     {
         Required("specification"): INSPECTION_SPECIFICATION_SCHEMA,
-        Required("created"): INSPECTION_CREATED_SCHEMA,
-        Required("build_log"): INSPECTION_BUILD_LOG_SCHEMA,
+        Required("created"): str,
```

Comment:
datetime?

Reply:
It's actually a string when we use the function https://github.com/thoth-station/amun-client/blob/444516e2c8df6d7e7a7caed14af47b316fd161a7/amun/lib.py#L98